assets/add_new_asset.md

```diff
@@ -180,10 +180,13 @@ Smart contract address update procedure:
 
 ### Image Requirements
 - File location: must be placed in the correct folder and subfolder within the [folder structure](#repository-structure).
-- File extension: `png`. Uppercase `PNG` is considered invalid.
-- File name：`logo.png`, all lowercase.
-- Dimension: `256 x 256 pixels` or `512 x 512 pixels`.
-- Background: Avoid transparency, set background as your brand color or white by default otherwise your token image wont look good on white/dark theme design.
+- File name：`logo.png`, all lowercase.  Extension: `png` (uppercase `PNG` is considered invalid).
+- Dimension: recommended `256 x 256 pixels`, maximum `512 x 512 pixels`, aspect ratio should be 1:1.
+- Edges and background:
```

Comment:
This part does not look well

<img width="1046" alt="Screen Shot 2021-09-16 at 6 17 36 AM" src="https://user-images.githubusercontent.com/1641795/133619169-ae2edc20-5648-404e-8e00-f033720756a2.png">